### ISSUE TITLE
Newsletters don't refresh without login to web UI

### ISSUE THREAD

**Issue by quad** [samuelclay/NewsBlur#1821]
I'm using NetNewsWire to access Newsblur.

Newsletters will stop refreshing without me periodically logging in via a browser to newsblur.com.

**Comment by samuelclay** [comment]
I wish NetNewsWire would hit the /reader/refresh_feeds endpoint, as that refreshes the login date and keeps your feeds current.

**Comment by samuelclay** [comment]
Here's the code that needs to be hit to ensure you're still a current user:
https://github.com/samuelclay/NewsBlur/blob/master/apps/profile/middleware.py#L22-L25

But it looks like NNW is hitting that:
https://github.com/Ranchero-Software/NetNewsWire/blob/main/SyncClients/NewsBlur/Sources/NewsBlur/NewsBlurAPICaller.swift#L77

**Comment by quad** [comment]
Yeah, the weird part is that feeds work normally. It’s only newsletters that stop refreshing after several days.

I tried tracing the newsletters refresh job but couldn’t see if something was triggering it that might be missed by NNN.

**Comment by quad** [comment]
@samuelclay Here's a video showing the issue…

https://github.com/samuelclay/NewsBlur/assets/24025/7d42a139-2b01-4dde-9096-90de3000b1a8

**Comment by quad** [comment]
My new theory is that the newsletter (and other) feeds are `needs_unread_recalc = True`, and [this code](https://github.com/samuelclay/NewsBlur/blob/db30d559a08a3d7014668a3c29e765fd0560d4db/apps/reader/views.py#L339-L343) is preventing the unread counts from being updated.

**Comment by quad** [comment]
Two proposals:

1. https://github.com/samuelclay/NewsBlur/pull/1871 stops letting the Android client get away crime 😉 
2. https://github.com/samuelclay/NewsBlur/pull/1872 normalises status quo … and then I can open a PR on NNW to set `update_counts` to `true`

**Comment by quad** [comment]
> I wish NetNewsWire would hit the /reader/refresh_feeds endpoint, as that refreshes the login date and keeps your feeds current.

What's the difference between the `load_feeds` and `refresh_feeds` endpoint? That is, could the cached/rate-limited aspects of `refresh_feeds` be internalised to `load_feeds?
